packages/core/src/components/FormLabel/FormLabel.jsx

```diff
@@ -52,8 +52,16 @@ export class FormLabel extends React.PureComponent {
   render() {
     const { fieldId, id, children } = this.props;
     const ComponentType = this.props.component;
-    const labelTextClasses = classNames(this.props.labelClassName);
-    const classes = classNames('ds-c-label', this.props.className);
+    const labelTextClasses = classNames(
+      'ds-c-label-content__container',
```

Comment:
This is a good solution for the choice labels, but since it affects _all_ labels, I wonder if an alternative could be adding `flex-basis: 100%` to `.ds-c-choice + label .ds-c-field-hint` instead so we can selectively apply this style only when it's needed?